### DISCUSSION TITLE
What's in the Octoverse 2023? DEV2781S

### DISCUSSION THREAD

**Discussion by martinwoodward** [community/community#74936] [Other Feature Feedback, Questions, & Ideas]
Hey folks 👋 - I'm getting ready to go onto the [Livestream](https://githubuniverse.com/) to talk you through this years [State of the Octoverse](https://github.blog/2023-11-08-the-state-of-open-source-and-ai/) report.  If you've got questions about the report or what your favorite areas are that you'd like to dig into then let me know!!!

**Comment by samus-aran** [comment]
If you want to see a recap jump to the [discussion announcement -> click here!](https://github.com/orgs/community/discussions/74740)
